test/powershell/engine/Remoting/SSHRemotingCmdlets.Tests.ps1

```diff
@@ -0,0 +1,63 @@
+##
+## SSH Remoting cmdlet tests
+##
+
+Describe "SSHTransport switch parameter value" -Tags 'CI' {
+
+    BeforeAll {
+
+        $TestCasesSSHTransport = @(
+            @{scriptBlock = {New-PSSession -HostName localhost -UserName UserA -SSHTransport:$false}; testName = 'New-PSSession SSHTransport parameter cannot have false value'}
+            @{scriptBlock = {Enter-PSSession -HostName localhost -UserName UserA -SSHTransport:$false}; testName = 'Enter-PSSession SSHTransport parameter cannot have false value'}
+            @{scriptBlock = {Invoke-Command -ScriptBlock {"Hello"} -HostName localhost -UserName UserA -SSHTransport:$false}; testName = 'Invoke-Command SSHTransport parameter cannot have false value'}
+        )
+    }
+
+    BeforeEach {
+        [System.Management.Automation.ErrorRecord] $paramError = $null
+    }
+
+    It "<testName>" -TestCases $TestCasesSSHTransport {
+
+        param($scriptBlock)
+        try
+        {
+            & $scriptBlock
+            throw "Parameter argument should not be valid"
+        }
+        catch { $paramError = $_ }
+
+        $paramError.FullyQualifiedErrorId | Should Match "ParameterArgumentValidationError"
+    }
+}
+
+Describe "SSHConnection parameter hashtable error conditions" -Tags 'CI' {
```

Comment:
Fixed.  Functional tests will be added when SSH remoting can be enabled via Enable-SSHRemoting.  Currently it is a manual process.  See RFC.